scripts/create_all_packages.sh

```diff
@@ -0,0 +1,70 @@
+# If Travis is running the script, let's cd in the right directory
+if [ ! -z ${TRAVIS_BUILD_DIR+x} ]; then
+    cd $TRAVIS_BUILD_DIR/scripts
+fi
+echo "Running from $(pwd)"
+
+# Set the secret environment variables if available
+# The file set_secret_environment_variables.sh should not be commited for security reasons
+# It is only useful to run the scripts locally.
+# Travis injects the same environment variables by itself
+if [ -r "set_secret_environment_variables.sh" ]; then
+  . "set_secret_environment_variables.sh"
+fi
+
+# For Travis : Decrypt the secret files
+if [ ! -f "./travisci_builder_id_key" ]; then
+    openssl aes-256-cbc -K $encrypted_3275b0f28807_key -iv $encrypted_3275b0f28807_iv -in secret_files.tar.gz.enc -out secret_files.tar.gz -d
+    tar xvzf ./secret_files.tar.gz
+fi
+
+# Use the first argument as a version number, else use the Travis tag, else use the commit id
+VERSION_TO_REPLACE="2\.0-beta"
+VERSION_PREFIX="2.0.0"
+if [ $# -ge 1 ]; then
+    VERSION=$1
+    echo "Packaging version $VERSION because it has been passed as an argument"
+elif [ ! "${TRAVIS_TAG}zz" == "zz" ]; then
+    echo "Debug : use the TRAVIS_TAG as the version number : $TRAVIS_TAG"
+    VERSION=$TRAVIS_TAG
+    echo "Packaging version $VERSION because it is the tag passed by Travis"
+else
+    COMMIT_ID=$(git rev-parse --short HEAD)
+    VERSION="${VERSION_PREFIX}commit-${COMMIT_ID}"
+    echo "Packaging version $VERSION"
+fi
+
+# Copy only the necessary files in a temporary directory
+mkdir -p ../tmp
+rm -rf ../tmp/*
+cp -r ../www ../webextension ../manifest.json ../manifest.webapp ../LICENSE-GPLv3.txt ../service-worker.js ../README.md ../tmp/
+
+# Replace the version number everywhere
+# But Chrome would only accept a numeric version number : if it's not, we only use the prefix in manifest.json
+regexpNumericVersion='^[0-9\.]+$'
+if [[ $VERSION =~ $regexpNumericVersion ]] ; then
+   sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" ../tmp/manifest.json
+else
+   sed -i -e "s/$VERSION_TO_REPLACE/$VERSION_PREFIX/" ../tmp/manifest.json
+fi
+sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" ../tmp/manifest.webapp
+sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" ../tmp/www/index.html
+
+mkdir -p ../build
+rm -rf ../build/*
+# Package for Chromium/Chrome
+./package_chrome_extension.sh $VERSION
+# Package for Firefox and Firefox OS
+# We have to put the real version string inside the manifest.json (which Chrome might not have accepted)
+# So we take the original manifest again, and replace the version inside it again
+cp ../manifest.json ../tmp/
+sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" ../tmp/manifest.json
+./package_firefox_extension.sh $VERSION
+./package_firefoxos_app.sh $VERSION
+
+CURRENT_DATE=$(date +'%Y-%m-%d')
+# Upload the files on download.kiwix.org
+echo "Uploading the files on http://download.kiwix.org/nightly/$CURRENT_DATE/"
+scp -r -p -i ./travisci_builder_id_key ../build/* nightlybot@download.kiwix.org:/var/www/download.kiwix.org/nightly/$CURRENT_DATE
```

Comment:
No, because the travisci_builder_id_key (which is necessary for the SSH connection) is not on the repo in clear : it is encrypted with the Travis way to do it